server/routes/users.js

```diff
@@ -90,12 +95,12 @@ router.delete('/user/delete/:id', async(req, res)=>{
     res.json({ message: 'User deleted successfully', deletedUser });
   } catch (error) {
     console.log(error)
-    res.status(500).send({message: 'Internal server error'});//Error handling
+    res.status(500).send({message: error.message});//Error handling
```

Comment:
Is the error message more readable if you just spit out error.message? Actually have never tried this so I'm just curious what it looks like

Reply:
Yea it is, otherwise it just says "Internal server error" on Postman instead of the actual error